### PR TITLE
Upgrade instructions for Toolkit in Docker

### PR DESCRIPTION
**Pull request by jfjoly**
# Description

Change the instructions for Docker update to use the TimescaleDB HA that includes Toolkit. The install instructions already point to the TimescaleDB HA image. https://docs.timescale.com/install/latest/installation-docker/#install-self-hosted-timescaledb-from-a-pre-built-container

# Links

N/A

# Review checklists
Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

- [ ] Is the content technically accurate?
- [ ] Is the content complete?
- [ ] Is the content presented in a logical order?
- [ ] Does the content use appropriate names for features and products?
- [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

- [ ] Is the content free from typos?
- [ ] Does the content use plain English?
- [ ] Does the content contain clear sections for concepts, tasks, and references?
- [ ] Are procedure and highlight tags used appropriately?
- [ ] Has the index been updated appropriately?
- [ ] Have any images been uploaded to the correct location, and are resolvable?
- [ ] Are all links provided in reference style, and resolvable?
- [ ] If the page index was updated, are redirects required
      and have they been implemented?
- [ ] Have you checked the built version of this content?

### CODE REVIEW

File: timescaledb/how-to-guides/update-timescaledb/updating-docker.md [hunk 1]
```diff
@@ -9,13 +9,13 @@ The following instructions assume that your docker instance is named
...
-Install the current TimescaleDB 2.0 image:
+Install the current TimescaleDB image:
 
 ```bash
-docker pull timescale/timescaledb:2.0.0-pg12
+docker pull timescale/timescaledb-ha:pg14-latest
 ```
 <highlight type="tip">
-If you are using PostgreSQL 11 images, use the tag `2.0.0-pg11`.
+If you are using another PostgreSQL version, look for the relevant tag on https://hub.docker.com/r/timescale/timescaledb-ha/tags.
```
Comment (charislam) [inline timescaledb/how-to-guides/update-timescaledb/updating-docker.md:18]:
```suggestion
If you are using another PostgreSQL version, look for the relevant tag in the
[Timescale HA Docker Hub repository](https://hub.docker.com/r/timescale/timescaledb-ha/tags).
```

**Review by charislam — APPROVED** [review]

**Comment by charislam** [comment]
The action "check Markdown links" is failing because it's reading the old anchors on the headers as links. We don't use that method of creating anchors anymore, so you can delete them to merge. (Or you might have enough permissions to ignore the failing check and merge.)

**Review by WireBaron — APPROVED** [review]

**Comment by Loquacity** [comment]
Oh no. I've just merged the Upgrades rework, and I'm afraid I might have broken everything 😢

**Comment by Loquacity** [comment]
@jfjoly I think we're going to need to rewrite. @charislam and/or I will get it done as soon as we get back from offsite.

**Comment by charislam** [comment]
@Loquacity @jfjoly I rebased and remade the changes on top of the new upgrade docs. Just need a review and this is good to merge

File: timescaledb/how-to-guides/upgrades/upgrade-docker.md [hunk 2]
```diff
@@ -69,7 +73,10 @@ data.
 
 ### Upgrading TimescaleDB within Docker
 
-1.  Pull the latest TimescaleDB image:
+1.  Pull the latest TimescaleDB image. The following command pulls the image for
```
Comment (Loquacity) [inline timescaledb/how-to-guides/upgrades/upgrade-docker.md:76]:
```suggestion
1.  Pull the latest TimescaleDB image. This command pulls the image for
```

File: timescaledb/how-to-guides/upgrades/upgrade-docker.md [hunk 3]
```diff
@@ -120,4 +129,15 @@ data.
     ALTER EXTENSION timescaledb UPDATE;
     ```
 
+1.  Update the [TimescaleDB Toolkit][toolkit] extension. Toolkit is packaged
+    with TimescaleDB's HA Docker image, and includes additional hyperfunctions
+    to help you with queries and data analysis.
```
Comment (Loquacity) [inline timescaledb/how-to-guides/upgrades/upgrade-docker.md:134]:
```suggestion
    to help you with queries and data analysis:
```

**Pull request merged**